patches/server/0010-Adventure.patch

```diff
@@ -1621,6 +1623,24 @@ index 0000000000000000000000000000000000000000..c10c0ffa29332d73328f088935a0b914
 +    public static @Nullable ChatFormatting asVanilla(final TextColor color) {
 +        return ChatFormatting.getByHexValue(color.value());
 +    }
++
++    // Style
++
++    public static net.minecraft.network.chat.Style asVanilla(Style style) {
++        net.minecraft.nbt.Tag encoded = Util.getOrThrow(AdventureCodecs.STYLE_MAP_CODEC.codec()
++            .encodeStart(net.minecraft.nbt.NbtOps.INSTANCE, style), IllegalStateException::new);
++
++        return Util.getOrThrow(net.minecraft.network.chat.Style.Serializer.CODEC
++            .parse(net.minecraft.nbt.NbtOps.INSTANCE, encoded), IllegalStateException::new);
++    }
++
++    public static Style asAdventure(net.minecraft.network.chat.Style style) {
++        net.minecraft.nbt.Tag encoded = Util.getOrThrow(net.minecraft.network.chat.Style.Serializer.CODEC
++            .encodeStart(net.minecraft.nbt.NbtOps.INSTANCE, style), IllegalStateException::new);
++
++        return Util.getOrThrow(AdventureCodecs.STYLE_MAP_CODEC.codec()
++            .parse(net.minecraft.nbt.NbtOps.INSTANCE, encoded), IllegalStateException::new);
++    }
```

Comment:
I think we are preferring to use JavaOps.INSTANCE for this instead of NbtOps.INSTANCE, as it just converts to a native java map instead of an nbt tag.